.pre-commit-config.yaml

```diff
@@ -24,6 +24,5 @@ repos:
     hooks:
       - id: mypy
         additional_dependencies: [types-requests]
-        language_version: python3.8
```

Comment:
This has been resolved in [72838bd](https://github.com/AutoResearch/autora/pull/271/commits/72838bde1816df0b4aaaca3609f07875caa7b4ca)